lte/gateway/python/scripts/session_manager_cli.py

```diff
@@ -147,8 +148,8 @@ def send_policy_rar(client, args):
                 match=FlowMatch(
                     direction=flow_direction,
                     ip_proto=ip_protocol,
-                    ipv4_src=flow_fields[2],
-                    ipv4_dst=flow_fields[4],
+                    ip_src=IPAddress(version=IPAddress.IPV4, address=flow_fields[2].encode('utf-8')),
+                    ip_dst=IPAddress(version=IPAddress.IPV4, address=flow_fields[4].encode('utf-8')),
```

Comment:
Please verify the 4G flows. It's a common code.

Reply:
I will check it with srsRAN setup and update on Monday.